src/pages/_app.js

```diff
@@ -1,5 +1,5 @@
 import Layout from "../components/Layout";
```

Comment:
If the `Layout` component doesn't exit anymore, it'll break. Replace it with some "Hello, World" content for now.